### PR TITLE
Fix flaky ClientDisconnect freb tests.

### PR DESCRIPTION
**Pull request by jkotalik**
For https://github.com/aspnet/IISIntegration/issues/1487

### CODE REVIEW

File: test/Common.FunctionalTests/Inprocess/FrebTests.cs [hunk 1]
```diff
@@ -96,23 +110,32 @@ private async Task<IISDeploymentResult> SetupFrebApp()
             return result;
         }
 
-        private void AssertFrebLogs(IISDeploymentResult result, params string[] data)
+        private void AssertSingleFrebLog(IISDeploymentResult result, params string[] data)
```
Comment (pakrym) [inline test/Common.FunctionalTests/Inprocess/FrebTests.cs]: Load all logs into an object structure and assert against it.

Comment (pakrym) [reply test/Common.FunctionalTests/Inprocess/FrebTests.cs]: Add a class FrebLog with FrebLogItem[] inside. Then you can use Assert.Contains and friends.

Comment (jkotalik) [reply test/Common.FunctionalTests/Inprocess/FrebTests.cs]: Create a FrebLogItem class, but didn't add a wrapper class.

**Comment by jkotalik** [comment]
Test failure: #1447

**Review by pakrym — APPROVED** [review]

**Comment by jkotalik** [comment]
Test failure #1447

**Pull request merged**